.gdn/r/pkg/portal/ssh/proxy_test.go.gdnresults

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Does all of this stuff really need to be checked in?  Even the .gdnresults files?  Not a .gitignore rule?
The files under /vendor will surely get deleted all the time when we do `go mod vendor`

Reply:
Following guidelines from what we could access on the Guardian documentation.  We could certainly try removing these child directories and see what happens. (they already include a .gitignore for the directory)

Reply:
Removed all subfolders, retested, and it worked. Committed changed to this PR.